File: 242. Valid Anagram.py
# Runtime: 40 ms, faster than 85.82% of Python3 online submissions for Valid Anagram.
# Memory Usage: 14.9 MB, less than 19.33% of Python3 online submissions for Valid Anagram.

class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        return sorted(s) == sorted(t)

File: 242. Valid Anagram.rs
impl Solution {
    pub fn is_anagram(s: String, t: String) -> bool {
        let mut s = s.chars().collect::<Vec<char>>();
        let mut t = t.chars().collect::<Vec<char>>();
        s.sort();
        t.sort();
        return s == t;
    }
}
